src/main/scala/diplomaticobjectmodel/HasLogicalTreeNode.scala

```diff
@@ -0,0 +1,9 @@
+package diplomaticobjectmodel
```

Comment:
This should match all the other packages and be
`package freechips.rocketchip.diplomaticobjectmodel`

Reply:
fixed